### PR TITLE
Documentation proof-reading

### PR DESCRIPTION
**Pull request by 14thibea**
I just checked the documentation and modified things which did not match the current command line.

### CODE REVIEW

File: docs/Interpret.md [hunk 1]
```diff
@@ -43,36 +43,27 @@ where:
...
+    - `--batch_size` (int) is the size of the batch used in the DataLoader. Default: `2`.
 - **Model selection**
-    - `--selection_metrics` (list of str) corresponds to the metrics according to which the 
-    [best models](Train/Details.md#model-selection) of `INPUT_MAPS_DIRECTORY` will be loaded. 
-    Choices are `best_loss` and `best_balanced_accuracy`. Default: `best_loss`.
+    - `--selection_metrics` (List[str]) is a list of metrics to find the best models to evaluate.
+      Default will predict the results for best model based on the loss only.
 - **Data management**
-    - `--participants_tsv` (str) is a path to a directory containing one TSV file per diagnosis
+    - `--participants_tsv` (path) is a path to a directory containing one TSV file per diagnosis
```
Comment (mdiazmel) [inline docs/Interpret.md]: Formally, the data type is `Path` (with P in caps) because it is a class.

**Review by mdiazmel — COMMENTED** [review]
Nice! 

A small comment on the argument data type, I thing that it should be `Path` and not `path` (to change everywhere).

File: docs/Interpret.md (hunk 1 above)
Comment (14thibea) [reply docs/Interpret.md]: Done in https://github.com/aramis-lab/clinicadl/pull/216/commits/388a7d135e6653ba4bf82dd767f5331fb57c75fb

**Comment by mdiazmel** [comment]
I'll merge this PR.

**Pull request merged**